Sources/Controllers/BasicAudioController.swift

```diff
@@ -143,30 +187,23 @@ internal class AudioController: NSObject {
         }
     }
 
+    private func createAudioPlayer(url: URL) -> AVAudioPlayer? {
```

Comment:
This can be removed and where needed use:
```
guard let audioPlayer = audioPlayer = try? AVAudioPlayer(contentsOf: url) else { return }
```
Since if we can't create the player we just don't play anything rather than returning nil and going about as if we did create a player